_pages/docs/merging-options.md

```diff
@@ -137,6 +147,20 @@ configuration. The `function` attribute is the localname of the function that
 will be called. This element may also have an `at` attribute, indicating where
 to find the source code for this function, and a `namespace` attribute.
 
+A `std-algorithm` element will allow you to configure options for the standard algorithm. Supported options are:
+
+#### Timestamp
+
+The timestamp config informs Smart Mastering which element to use for sorting. When merging, the values are sorted in recency order from newest to oldest based on this timestamp. If the timestamp is not provided then there is no recency sort.
+
+```
+  <std-algorithm xmlns:es="http://marklogic.com/entity-services" xmlns:sm="http://marklogic.com/smart-mastering">
+    <timestamp path="/es:envelope/es:headers/sm:sources/sm:source/sm:dateTime" />
+  </std-algorithm
+```
+
+Note that any namespaces used in the @path attribute must be defined on the <std-algorithm> element.
```

Comment:
What if the path includes an element with the "" namespace? Please add a note like, "The default namespace for evaluating the path is the empty namespace." (or whatever it's called)